src/main/java/uk/gov/hmcts/ccd/data/casedetails/JurisdictionMapper.java

```diff
@@ -15,6 +15,10 @@ public JurisdictionDisplayProperties toResponse(Jurisdiction jurisdiction) {
         result.setName(jurisdiction.getName());
```

Comment:
would it be possible to properly model the banner in Object Oriented terms with a Banner object?
Furthermore, but this is my personal opinion, I wouldn't have added a banner directly into a Jurisdiction. I would have had reverted the association, i.e. kept it as a separate object and have each Banner refer to a JurisdictionId.

Reply:
Done changes